package/rtcm3_in_protocol/src/Makefile

```diff
@@ -0,0 +1,26 @@
+TARGET=librtcm3_in_protocol
+SOURCES=info_rtcm3.c framer_rtcm3_in.c
```

Comment:
see note on renaming to match `info_rtcm3_out.c` naming convention

Reply:
good catch, done